roles/ssh_jumper/templates/ssh_host.conf.j2

```diff
@@ -0,0 +1,11 @@
+Host {{ (cifmw_ssh_jumper_config.patterns | default([]) + [cifmw_ssh_jumper_config.hostname]) | join(' ') }}
+    Hostname {{ cifmw_ssh_jumper_config.hostname }}
```

Comment:
`HostName`

Reply:
I don't think ssh is that strict actually - though we can take the opportunity to converge to CamelCase.

Here's the currently generated content:
```
Host crc-0 crc-0.builder2 cifmw-crc-0 192.168.100.30
  ProxyJump virtuser@builder2.rh.lain.internux.ch
  Hostname 192.168.100.30
  User core
  StrictHostKeyChecking no
  UserKnownHostsFile /dev/null
```

BTW: we'll need to advertise "clean before trying to redeploy", else ppl will end with some fancy issues.